components/ChannelProfile/ChannelProfile.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Modal, View, Image, TouchableOpacity } from 'react-native';
+import { Text, Icon, Card, CardItem } from 'native-base';
+import { Font, AppLoading } from "expo";
+import AutoLink from 'react-native-autolink'
+import _ from 'lodash';
+
+import styles from '../../components/UserProfile/UserProfileStyle';
+import {getProfilePicture} from '../../helpers/imageCache'
+
+export default class ChannelProfile extends React.Component {
+
+  onClose = () => {
+    const { onClose } = this.props;
+    onClose();
+  }
+
+  getCloseJSX() {
```

Comment:
Personally I would just put this JSX right in the render function since there isn't any additional logic being done to generate the jsx